app/Http/Controllers/HomeController.php

```diff
@@ -14,13 +17,44 @@ public function __construct(Request $request) {
         $this->middleware('auth');
     }
 
-    public function index() {
-        return view('home');
+    public function redirect(Request $request)
+    {
+        $dashboard = Dashboard::where('user_id', $request->user()->user_id)->first();
+        if (empty($dashboard->dashboard_id))
+        {
+            $dashboard = new dashboard;
+            $dashboard->user_id        = $request->user()->user_id;
+            $dashboard->dashboard_name = 'Default';
+            $dashboard->access         = 0;
+            $dashboard->save();
```

Comment:
You don't have to set user_id and you can user user directly.
$request->user()->dashboards()->save($dashboard);